dependencies.props

```diff
@@ -59,7 +59,7 @@
     <XmlUpdateStep Include="CoreFx">
       <Path>$(MSBuildThisFileFullPath)</Path>
       <ElementName>CoreFxVersion</ElementName>
-      <PackageId>Microsoft.Private.CoreFx.NETCoreApp</PackageId>
+      <PackageId>Microsoft.Private.CoreFx.UAP</PackageId>
```

Comment:
I don't understand why this would change anything... aren't these packages always built with the same version?
https://github.com/dotnet/versions/blob/8d7763e82daa7cf6afd4abcb75b1f48be5e2e83d/build-info/dotnet/corefx/master/Latest_Packages.txt#L5-L6

I think `CoreFxVersion` should be renamed to something more specific if they aren't.